video/out/vulkan/utils.c

```diff
@@ -328,6 +329,27 @@ bool mpvk_find_phys_device(struct mpvk_ctx *vk, const char *name, bool sw)
     return false;
 }
 
+bool mpvk_get_phys_device_uuid(struct mpvk_ctx *vk, uint8_t *uuid_out)
```

Comment:
As a slight cosmetic change, could this be changed to `uint8_t uuid_out[VK_UUID_SIZE]` to document the size requirements?